### ISSUE TITLE
[Data] CNN Architecture & Training

### ISSUE THREAD

**Issue by MeralYavuzturk** [MeralYavuzturk/Smart_Farming-#7]
- [ ] Assigned: @Mrvzcn  & @AyseMutluay 
- [x] Build CNN layers (Conv2D, Pooling).
- [x] Determine hyperparameters (Learning rate, Epoch count).
- [ ] Start model training.
- [ ] Apply Data Augmentation to improve success rate.

**Comment by AyseMutluay** [comment]
I built the CNN architecture using Conv2D and Pooling layers.
I used BatchNormalization in the model.
I determined the hyperparameters (learning rate and epoch count).
I set learning rate to 0.001 and epochs to 30.
Note: I performed all operations in Google Colab.
